node_exporter.go

```diff
@@ -172,6 +172,15 @@ func main() {
 	}
 	go func() {
 		http.Handle("/metrics", handler)
+		http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+			w.Write([]byte(`<html>
+                       <head><title>Node Exporter</title></head>
+                       <body>
+                       <h1>Node Exporter</h1>
+                       <p><a href='/metrics'>Metrics</a></p>
```

Comment:
Missed this in the haproxy one, but change the single quotes to double ones (the common way in HTML).